lib/tenderjit/runtime.rb

```diff
@@ -367,6 +367,7 @@ def call_cfunc func_loc, params
       raise NotImplementedError, "too many parameters" if params.length > 6
       raise "No function location" unless func_loc > 0
 
+      push_reg REG_BP # alignment
```

Comment:
In this case do `@fisk.push(REG_BP)`.

The "runtime" DSL isn't being used everywhere yet (though I would like to use it in more places)